code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -3,7 +3,6 @@
 /obj/item/device/mmi/digital/New()
 	src.brainmob = new(src)
 	src.brainmob.stat = CONSCIOUS
-	src.brainmob.add_language("Robot Talk")

```

Comment:
Will this prevent gibbed/broken cyborgs from being able to use binary? I don't _think_ it would, but it never hurts to double-check.

Reply:
That is on purpose, there is a component that regulates binary on cyborg's body, why would the brain have it already.

Reply:
I concur with Albermeme. There's also the fact that MMIs are not lawbound makes this iffy.